test/certUtils.test.js

```diff
@@ -442,7 +442,7 @@ describe("cert utils", function() {
 			it("decodes all transport types", function() {
 				const ret = resolveOid(
 					"1.3.6.1.4.1.45724.2.1.1",
-					new Uint8Array([0x03, 0x02, 0x03, 0xF8]).buffer
+					0xF8
```

Comment:
Here you are changing the way the method is called - is this method provided to end users or only exposed during tests? We need to make sure that we are not breaking existing code for users.

Reply:
Will need to have a closer look om that. It is a fully internal function working against pkijs (which has changed it's internals from 2 to 3). But it could be exposed indirectly. I'll get back later

Reply:
This is a tricky one... It is exported, pretty much everything is exported - but just to be able to test the `dist/`. You can call it an undocumented feature.

Reply:
I would say that it's a fully internal function. Also undocumented. So a patch? 

But if we want to be strict about semver, it's a minor... 🤔 

Anyway, by learning from this i added jsdoc access level to all exported functions of KeyUtils...

Reply:
Thanks for the clarification! If its not directly from our package and is likely not used, it is good enough, given that we just recently did a major version and people will slowly start migration.